A36-xds-for-servers.md

```diff
@@ -334,6 +339,44 @@ connection, it must still be checked when verifying that the most-specific
 matching logic is guaranteed to produce a unique result during Listener
 validation.
 
+### Resource Validation
+
+The `Listener` proto validation rules are modified as follows:
+
+ - The `listener_filters` field must be empty.
+ - The `use_original_dst` field must not be set with the value `true`.
+ - For every entry in the `filter_chains` field:
+   - The `filter_chain_match` field must be normalized as described in the
+     [FilterChainMatch section](#filterchainmatch) and there must not be
+     duplication between any normalized matchers in the `Listener`.
+ - For the `default_filter_chain` field and every entry in the `filter_chains`
+   field:
+   - The `filters` field must have exactly one entry. Inside of it:
+     - The `type_url` field must have the value
+       `envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager`
+     - Inside the `value` field, decoded as a `HttpConnectionManager` message:
+       - The `http_filters` field must be validated as specified in
```

Comment:
As per my comment elsewhere, I think that these two bullets have the same conditions: if the implementation supports HTTP filters, then it must do both; if it does not, then it can do neither.  So I suggest saying something like this:

- The `value` field must contain a valid `HttpConnectionManager` message.  If the implementation supports HTTP filters, then the following must be validated:
  - The `http_filters` field must be validated as per A39.
  - Either the `route_config` field or the `rds` field must be set:
    - If `route_config` is set, it must be validated according to the `RouteConfiguration` validation rules below.
    - If `rds` is set, an RDS watch must be started for the specified resource.